internal/zinc-scripted/src/main/contraband/scripted.contra

```diff
@@ -0,0 +1,14 @@
+package sbt.internal.inctest
```

Comment:
If this is just to test zinc (properly, with multi-projects), can we please avoid using contraband for this?